modules/contacts/contacts.service.js

```diff
@@ -1,20 +1,66 @@
 const Contact = require('./contacts.model');
+const {
+  CONTACT_NOT_FOUND,
+  CONTACT_ALREADY_EXIST,
+} = require('../../error-messages/contacts.messages');
 
 class ContactsService {
   async getContacts() {
     return await Contact.find();
   }
 
+  async getContactById(id) {
+    const contact = await Contact.findById(id);
```

Comment:
It is better to not throw an error here but return existing contact or empty value it to the resolver and return an error there